### PR TITLE
ProxySQL Doc

### PR DESCRIPTION
**Pull request by TasdidurRahman**
Signed-off-by: TasdidurRahman <tasdid@appscode.com>

### CODE REVIEW

File: docs/guides/elasticsearch/quickstart/overview/index.md [hunk 1]
```diff
@@ -175,7 +175,7 @@ Here,
 Let's create the Elasticsearch CR that is shown above:
 
 ```bash
-$ kubectl apply -f https://github.com/kubedb/docs/raw/{{< param "info.version" >}}/docs/guides/elasticsearch/quickstart/overview/yamls/elasticsearch.yaml
```
Comment (spectro30) [inline docs/guides/elasticsearch/quickstart/overview/index.md]: why `mysqlgrp` in elasticsearch?

File: docs/guides/elasticsearch/quickstart/overview/index.md [hunk 2]
```diff
@@ -531,7 +531,7 @@ Say, the Elasticsearch CR was deleted with `spec.terminationPolicy` to `Halt` an
 You can do it by simpily re-deploying the original Elasticsearch object:
 
 ```bash
-$ kubectl apply -f https://github.com/kubedb/docs/raw/{{< param "info.version" >}}/docs/guides/elasticsearch/quickstart/overview/yamls/elasticsearch.yaml
```
Comment (spectro30) [inline docs/guides/elasticsearch/quickstart/overview/index.md]: ?

File: docs/guides/mariadb/quickstart/overview/index.md [hunk 3]
```diff
@@ -79,7 +79,7 @@ spec:
 ```
 
 ```bash
-$ kubectl apply -f https://github.com/kubedb/docs/raw/{{< param "info.version" >}}/docs/guides/mariadb/quickstart/overview/examples/sample-mariadb.yaml
```
Comment (spectro30) [inline docs/guides/mariadb/quickstart/overview/index.md]: why mysqlgrp in MariaDB?

File: docs/guides/mysql/custom-rbac/index.md [hunk 4]
```diff
@@ -70,7 +70,7 @@ secrets:
 Now, we need to create a role that has necessary access permissions for the MySQL instance named `quick-mysql`.
 
 ```bash
-$ kubectl apply -f https://github.com/kubedb/docs/raw/{{< param "info.version" >}}/docs/guides/mysql/custom-rbac/yamls/my-custom-role.yaml
```
Comment (spectro30) [inline docs/guides/mysql/custom-rbac/index.md]: why `prx-custom-role` on mysql

File: docs/guides/proxysql/clustering/proxysql-cluster/index.md [hunk 5]
```diff
@@ -0,0 +1,544 @@
...
+
+First we need to create a script to sent load over the ProxySQL. We will use the test user and the test table to check and send the load.
+
+```bash
+$ kubectl exec -it -n demo proxy-server-1 -- bash
+root@proxy-server-1:/# apt update
+... ... ... 
+root@proxy-server-1:/# apt install nano
+... ... ... 
+root@proxy-server-1:/# nano load.sh
```
Comment (spectro30) [inline docs/guides/proxysql/clustering/proxysql-cluster/index.md:289]: Where is link of `load.sh` file?
I can see you have stored it in `examples` folder

File: docs/guides/proxysql/concepts/opsrequest/index.md [hunk 6]
```diff
@@ -0,0 +1,272 @@
...
+    - alias: server
+      subject:
+        organizations:
+        - kubedb:server
+      dnsNames:
+      - localhost
+      ipAddresses:
+      - "127.0.0.1"
+      emailAddresses:
+      - "md.alifbiswas@gmail.com"
```
Comment (spectro30) [inline docs/guides/proxysql/concepts/opsrequest/index.md]: use dummy email address, not mine. :crying_cat_face: 

Applicable for other occurrences.

**Review by spectro30 — CHANGES_REQUESTED** [review]

File: docs/guides/elasticsearch/quickstart/overview/index.md (hunk 1 above)
Comment (TasdidurRahman) [reply docs/guides/elasticsearch/quickstart/overview/index.md]: resolved

File: docs/guides/elasticsearch/quickstart/overview/index.md (hunk 2 above)
Comment (TasdidurRahman) [reply docs/guides/elasticsearch/quickstart/overview/index.md]: resolved

File: docs/guides/mariadb/quickstart/overview/index.md (hunk 3 above)
Comment (TasdidurRahman) [reply docs/guides/mariadb/quickstart/overview/index.md]: resolved

File: docs/guides/mysql/custom-rbac/index.md (hunk 4 above)
Comment (TasdidurRahman) [reply docs/guides/mysql/custom-rbac/index.md]: resolved

File: docs/guides/proxysql/concepts/opsrequest/index.md (hunk 6 above)
Comment (TasdidurRahman) [reply docs/guides/proxysql/concepts/opsrequest/index.md]: resolved

File: docs/guides/proxysql/clustering/proxysql-cluster/index.md (hunk 5 above)
Comment (TasdidurRahman) [reply docs/guides/proxysql/clustering/proxysql-cluster/index.md:289]: added file link

**Review by spectro30 — APPROVED** [review]
LGTM

**Pull request merged**